### PR TITLE
chat working with new setup

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**

### CODE REVIEW

File: src/components/Call/Chat.tsx [hunk 1]
```diff
@@ -75,21 +70,18 @@ const Chat: React.FC<Props> = ({ roomClient, inmateId, call }) => {
...
+    // need to first add status (success, status, pending)
+    // https://github.com/Ameelio/pathways-client/issues/32
+    roomClient
+      .request("textMessage", {
+        callId: call.id,
+        contents: draftMessage,
+      })
+      .then(
+        () => console.log("ssucces"),
+        (rejection: string) => console.log(rejection)
```
Comment (mpekala23) [inline src/components/Call/Chat.tsx:83]: niiiiccceeee

File: src/redux/modules/call.ts [hunk 2]
```diff
@@ -38,7 +38,8 @@ export const initializeVisit = createAsyncThunk(
     videoHandler: CallHandler;
   }) => {
     const socket = io.connect(
-      `https://localhost:${videoHandler.port}` || "localhost:8000"
+      `https://localhost:${videoHandler.port}` || "localhost:8000",
```
Comment (mpekala23) [inline src/redux/modules/call.ts:41]: is the `|| "localhost:8000"` doing anything here?

**Review by mpekala23 — APPROVED** [review]
looks good to me! tiny question but nothin crazy

File: src/redux/modules/call.ts (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/redux/modules/call.ts:41]: nope lol

**Pull request merged**